charts/internal/seed-controlplane/charts/csi-vsphere/templates/vpa-vsphere-csi-controller.yaml

```diff
@@ -1,15 +1,61 @@
 ---
-apiVersion: autoscaling.k8s.io/v1beta2
+apiVersion: autoscaling.k8s.io/v1
 kind: VerticalPodAutoscaler
 metadata:
   name: vsphere-csi-controller-vpa
   namespace: {{ .Release.Namespace }}
 spec:
   resourcePolicy:
     containerPolicies:
-      - containerName: '*'
-        minAllowed:
-          memory: 20Mi
+    - containerName: vsphere-csi-controller
+      minAllowed:
+        memory: 20Mi
+      maxAllowed:
+        cpu: {{ .Values.resources.controller.maxAllowed.cpu }}
+        memory: {{ .Values.resources.controller.maxAllowed.memory }}
+      controlledValues: RequestsOnly
+    - containerName: vsphere-csi-syncer
+      minAllowed:
+        memory: 20Mi
```

Comment:
are the `minAllowed` values here based on any actual measurement? Are we sure these containers cannot work correctly with less memory?
If not: let's stick to what the PR is about: adding `maxAllowed` to prevent workload from becoming unschedulable

Reply:
`minAllowed` value was set for all containers, so I just keep that setting

Reply:
Ah, sorry, I missed that the previous `"*"' matcher had a `minAllowed` configuration above. Keeping it makes sense then, yes!